docs/Rest-API-description.md

```diff
@@ -230,11 +232,12 @@ Response body on success:
 
 
 ### List of All Saved Examples of the Subject
+```since 1.0 version```
```

Comment:
![image](https://user-images.githubusercontent.com/3736126/149140837-58b872c2-4928-4301-a7b5-e446fe59d5d8.png)
Wrong place

Reply:
Please, delete it from here

Reply:
I deleted the text "Since 0.6 version" from table